debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -1130,6 +1130,7 @@ C|CFR_METHOD_ACCESS_NEWJDK5_MASK
 C|CFR_METHOD_EXT_HAS_CODE_TYPE_ANNOTATIONS
 C|CFR_METHOD_EXT_HAS_METHOD_TYPE_ANNOTATIONS
 C|CFR_METHOD_EXT_INVALID_CP_ENTRY
+C|CFR_METHOD_EXT_HAS_SCOPED_ANNOTATION
```

Comment:
I'm fairly certain this file is generated, so it should not be updated

Reply:
I don't see `C|CFR_METHOD_EXT_HAS_SCOPED_ANNOTATION` after building though

Reply:
Actually `openj9-openjdk-jdk16/build/linux-x86_64-server-release/vm/superset.dat` has it